crates/rspack_binding_values/src/stats.rs

```diff
@@ -794,6 +794,8 @@ impl From<rspack_core::StatsModuleIssuer<'_>> for JsStatsModuleIssuer {
 pub struct JsStatsModuleReason {
   #[napi(ts_type = "JsModuleDescriptor")]
   pub module_descriptor: Option<JsModuleDescriptorWrapper>,
+  pub resolved_module_descriptor: Option<JsModuleDescriptorWrapper>,
```

Comment:
Should attach `#[napi(ts_type = "JsModuleDescriptor")]` for generating right ts type